src/Turbo/TurboBundle.php

```diff
@@ -23,6 +25,15 @@
  */
 final class TurboBundle extends Bundle
 {
+    public const STREAM_FORMAT = 'turbo_stream';
+    public const STREAM_MEDIA_TYPE = 'text/vnd.turbo-stream.html';
+
+    public function boot()
+    {
+        // TODO: remove this hack when HttpFoundation 6.1 will be required https://github.com/symfony/symfony/pull/44980
+        (new Request())->setFormat(self::STREAM_FORMAT, self::STREAM_MEDIA_TYPE);
```

Comment:
This could be nice to allow bundles to register their own media types without having to rely on this hack. I don't know why `Request::setFormat()` isn't marked `static`.

Reply:
Actually, I'd be in favor of closing #44980 and removing the todo above.
The current API might not be ideal but it works.

Reply:
@dunglas maybe we can add a `@static` as phpdoc and make it really static then in symfony 7.